src/ireg.cpp

```diff
@@ -28,7 +22,7 @@ main( int argc, char **argv ) {
     int nArgv;
 
 
-    optStr = "D:fhkKCG:R:vVZ";
+    optStr = "D:fhkKCrR:vVZ";
```

Comment:
G is not processed in `parseCmdLineOpt` as it has to do with resource groups which were removed in 4.0.0. I figured it's about time to remove it. I can put it back if anybody disagrees

Reply:
torch approved.

Reply:
Sweet